app/controllers/charges_controller.rb

```diff
@@ -1,25 +1,50 @@
 class ChargesController < ApplicationController
 
   def new
+    @amount = (cart.total * 100)
   end
 
   def create
-  @amount = 500
+    @amount = (cart.total * 100)
 
-  customer = Stripe::Customer.create(
-    :email => 'example@stripe.com',
-    :card  => params[:stripeToken]
-  )
+    customer = Stripe::Customer.create(
+      :email => 'example@stripe.com',
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.